basic_c_tests/ptr-dereference3.c

```diff
@@ -6,13 +6,13 @@ void foo(int q){
 }
 int main(){
 
-int *s,*r,***x,**y,t,z,k;
+int *s,*r,**x,**y,t,z,k;
 	s = &t;
 	r = &z;
 	y = &r;
 	s = r;
   MUSTALIAS(s,&z);
-	x = *y;
+	x = y;
```

Comment:
This change is incorrect. x and r as indicated should be must alias here. Your change here will made x points to r.

Reply:
Oh yes, I had updated the test case such that **x must points to r** and not **x must alias r**.
I have updated the test case.